modules/client/src/client-common/types/common.ts

```diff
@@ -1,10 +1,13 @@
-export const SupportedNetworksArray = [
-  "homestead",
-  "goerli",
-  "matic",
-  "maticmum",
-] as const;
-export type SupportedNetwork = typeof SupportedNetworksArray[number];
+export enum SupportedNetwork {
+  MAINNET = "homestead",
+  GOERLI = "goerli",
+  POLYGON = "matic",
+  MUMBAI = "maticmum",
+}
+
```

Comment:
Will there be any difference in TypeScript checking? Will the strict "homestead" (etc) values remain strictly checked?

Reply:
Yes the main difference is that if you want to use the sting values I believe you need to use `"homestead" as SupportedNetwork`